internal/notification/service.go

```diff
@@ -237,7 +238,12 @@ func (s *service) Test(ctx context.Context, notification domain.Notification) er
 		agent = NewTelegramSender(s.log, notification)
 	}
 
-	g, ctx := errgroup.WithContext(ctx)
+	g, _ := errgroup.WithContext(ctx)
+
+	if agent == nil {
+		s.log.Error().Msgf("unsupported notification type: %v", notification.Type)
+		return errors.New("unsupported notification type")
+	}
```

Comment:
The idiomatic solution is to add a `default` case that return an error if it's an unknown or missing type.

Reply:
ouhh okay will update it

Reply:
updated it @zze0s let me know if that's what you wanted. 
ngl forgot about default in switch statement.